### PR TITLE
DLCL-832 

### PR DESCRIPTION
**Pull request by denfedweb**
https://opentechdev.atlassian.net/browse/DLCL-832

https://opentechdev.atlassian.net/browse/DLCL-833

### CODE REVIEW

File: src/components/basic/DlAlert/DlAlert.vue [hunk 1]
```diff
@@ -111,6 +121,16 @@ export default defineComponent({
         text: {
             type: String,
             default: null
+        },
+        darkMode: {
```
Comment (fadiDL) [inline src/components/basic/DlAlert/DlAlert.vue:125]: this is an incorrect approach in this case. adding a prop to define theme mode is not necessary.

File: src/assets/globals.scss
Comment (fadiDL) [inline src/assets/globals.scss:1]: You can push new variables for the info colors so example 

--dl-color-alert-success
--dl-color-alert-warn
--dl-color-alert-error
--dl-color-alert-info

(with their backgrounds of course )


Set them hardcoded and for them not to be overridden by the dark theme. that should achieve what you are looking for. then you can set the value of  `--dl-color-positive to var(--dl-color-alert-success)` for example

**Review by fadiDL — CHANGES_REQUESTED** [review]

File: src/components/basic/DlAlert/DlAlert.vue (hunk 1 above)
Comment (denfedweb) [reply src/components/basic/DlAlert/DlAlert.vue:125]: this is not a definition, this is turning on and off the dark mode for the alert
alert is used in toast message
in toast message colors should not change in dark mode, but in alert should

File: src/components/basic/DlAlert/DlAlert.vue [hunk 2]
```diff
@@ -111,6 +121,16 @@ export default defineComponent({
...
+            type: Boolean,
+            default: true
+        },
+        closeButtonPosition: {
```
[the remark this answers is not in the document]
Comment (denfedweb) [reply src/components/basic/DlAlert/DlAlert.vue:129]: in the alert it should be on top, in the toast message should be in the center

File: src/assets/globals.scss
Comment (denfedweb) [reply src/assets/globals.scss:1]: I made it so that using the class you can disable color changes for dark mode

**Comment by fadiDL** [comment]
I took it manually and updated the code and did the refactor. merging soon.

**Review by fadiDL — APPROVED** [review]

**Pull request merged**